projects/batfish-client/src/test/java/org/batfish/client/ClientTest.java

```diff
@@ -801,7 +801,7 @@ public void testListTestrigsInvalidParas() throws Exception {
 
   @Test
   public void testLoadQuestionsInvalidParas() throws Exception {
-    testInvalidInput(LOAD_QUESTIONS, new String[] {});
+    testInvalidInput(LOAD_QUESTIONS, new String[] {"path1", "path2"});
```

Comment:
add more tests for the different use cases. I want to know that all 3 cases work correctly.